fixtures/ui_coverage.py

```diff
@@ -181,9 +181,9 @@ def _gem_install():
         def _bundle_install():
             self.ipapp.ssh_client.run_command('yum -y install git')
             self.ipapp.ssh_client.run_command('cd {}; bundle'.format(rails_root))
+
         version.pick({
-            version.LOWEST: _bundle_install,
-            '5.4': _gem_install,
+            version.LOWEST: _gem_install,
```

Comment:
Q: please double check I suspect this version pick can be removed at all.

Reply:
Depends on which version after 5.4 started to use _bundle_install.
This version pick can be removed if _bundle_install was used from 5.8 version.
Since I don't have this information, I've left this as it is.